src/test/java/com/revature/controllers/QuestionControllerTests.java

```diff
@@ -60,5 +60,49 @@ public void testGetAllQuestionsHappyPath() throws Exception {
 			
 	}
 	
+	/* @Author ken */
+	@Test
+	public void testGetAllQuestionsByUserId() throws Exception {
+		
+		// Create page of data
+		List<Question> questions = new ArrayList<>();
+		questions.add(new Question(1,1,"title","content", LocalDate.MIN, LocalDate.MIN, true, 1));
+		Page<Question> pageResult = new PageImpl<>(questions);
+		
+		// Stub getAllQuestions to return page of data
+		when(questionService.getAllQuestionsByUserId(Mockito.any(Pageable.class), Mockito.anyInt())).thenReturn(pageResult);
+		
+		// Call API end point and assert result
+		mvc.perform(get("/questions/user/1")
+			.accept(MediaType.APPLICATION_JSON))
+			.andExpect(status().isOk())
+			.andExpect(content()
+					.contentTypeCompatibleWith(MediaType.APPLICATION_JSON))
+			.andExpect(jsonPath("$.content[0].id", is(1)));
+			
+	}
+	
+	/* @Author ken 
```

Comment:
Why is this test commented out? We could use @Ignore to not run the test, but if the test has no value then it should be removed. If it is failing, then the feature should be fixed such that the test passes.

Reply:
fixed